### PR TITLE
Improve accessibility of search components and documentation

### LINKED ISSUE #943: Improve documentation/accessibility of the search feature
**Please describe the feature**

As a user, I want to know what I can search on so that I am able to use quick and global search to their full capacity.

**Acceptance Criteria**

- [ ] The documentation is updated accordingly including linking to the global search page (there is already a keyword `cholesterol` that should be linked)
- [x] Maybe some kind link to the documentation from the quick search? Or highlight global search? 
- [x] The changes are tested on multiple models and browsers
- [x] there is no requirement on having a (new) general strategy for linking functionality/features to sections in Documentation (although improvement ideas are welcome, but they might end up as different issues)

**Additional context**

User should know that they can search on for example PMID

### PR DESCRIPTION
**Pull request by jonandernovella**
**Related issue(s) and PR(s)**  
This PR closes #943 

**Type of change**  

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other
 
**List of changes made**  

- Added component for connection documentation with web components
- Implemented this component in search and data overlay
- Minor accessibility improvements: role tags
- Added cypress test for new component

**Screenshot of the fix**  

image

**Testing**  

- npx cypress run cypress -b firefox --spec cypress/e2e/helpbutton.cy.js
- Visit quick search and global search pages. Also, check the data overlay page

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [x] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by perjo — APPROVED** [review]
Looks good to me!

**Comment by KattisLej** [comment]
Starting review

File: frontend/src/components/shared/HelpButton.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<template>
+  <button type="button" class="helpCircleButton" @click="redirectToPage()">
```
Comment (KattisLej) [inline frontend/src/components/shared/HelpButton.vue:2]: I checked the issue using the WAVE accessibility evaluation tool (https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) and it doesn't like the empty button, meaning the button has no text. Using:
aria-label='info button' 
inside button tag will fix that issue

**Review by KattisLej — CHANGES_REQUESTED** [review]
* I left one accessibility comment on the help button and using the WAVE plugin to do so.

* I think the info icon could be a little bit bigger to be more visible:
image
* Also the icon no longer has a cursor when hovering

* This is not from your changes but you could add this for accessebilty reason for improvement:
aria-label='search box' 
in the GemSearch.Vue inside the searchbox that begins with the input tag on row 21

Over all nice done, this is a very nice improvement

**Comment by jonandernovella** [comment]
>     * This is not from your changes but you could add this for accessebilty reason for improvement:
>       aria-label='search box'
>       in the GemSearch.Vue inside the searchbox that begins with the input tag on row 21

I think we already have done this with the role="searchbox"

**Review by KattisLej — APPROVED** [review]
Looks very nice and accessible :)

**Comment by Jessica-Sandler** [comment]
This change adds some useful functionality and looks nice, and as far as I can see everything works as it is supposed to do. Good work!

**Pull request merged**